pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -739,6 +739,14 @@ def test_eq(self):
 
         assert DateOffset(milliseconds=3) != DateOffset(milliseconds=7)
 
+    def test_milliseconds_combination(self):
+        """
```

Comment:
Thanks for the idea here but we don't typically include docstrings as part of tests - can you just add `GH #10525` as a comment?